src/codesandboxer/utils/getAllImports.js

```diff
@@ -4,7 +4,8 @@ import type { Import } from '../types';
 
 const getAllImports = (code: string): Array<Import> => {
   let imports = code.match(new RegExp(importPattern, 'g'));
-  if (imports) return imports.map(mpt => new RegExp(importPattern).exec(mpt));
+  if (imports)
+    return imports.map(mpt => new RegExp(importPattern).exec(mpt).map(a => a));
```

Comment:
What's the purpose of .map(a => a) in this context?

Reply:
`Regexp.exec` return an iterable that isn't an array. Mapping it transforms it into an array instead.

Reply:
awesome, then lgtm